pkg/target/manager.go

```diff
@@ -60,7 +60,7 @@ func NewManager(
 		logger:            logger,
 		reg:               reg,
 		externalLabels:    externalLabels,
-		ksymCache:         ksym.NewKsymCache(logger),
+		ksymCache:         ksym.NewKsymCache(logger, reg, externalLabels),
```

Comment:
rather than external labels, this should be the name of the service discovery config no?

Reply:
N00b question 😅 , but what would the name of the service discovery config be?

I might be wrong, but in the end, I don't think we really need any labels here, no?

Reply:
This is more accurate. These metrics are per manager.

Reply:
As far as I've seen, we only have one manager no? So we don't need labels to identify the manager (yet), no?

Reply:
I think so I need to double-check, in theory, there should only be one because systemd and kubernetes discovery, they're mutually exclusive